src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -194,7 +194,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
     register_cli_argument(scope, 'subnet_name', help='The subnet name.  Creates if creating a new VNet, references if referencing an existing VNet.')
     register_cli_argument(scope, 'admin_password', help='Password for the Virtual Machine if Authentication Type is Password.')
     register_cli_argument(scope, 'ssh_key_value', action=VMSSHFieldAction)
```

Comment:
You can apply `file-type` on this one. Even this could be the public key value, but `~` would not exist, so it is a safe no-op